### PR TITLE
Made com.hedera.pbj.runtime.ProtoConstants public

### PR DESCRIPTION
**Pull request by LLITCHEV**
Added methods to WritableSequentialData and ReadableSequentialData.

This resolves issue #44.

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by jasperpotts — CHANGES_REQUESTED** [review]
I am not sure what you did here. Adding the readTag() and writeTag() methods is not helpful. The issue description explained what needed to be done.

> Solution
>
>Make com.hedera.pbj.runtime.ProtoConstants constant fields public and
>make com.hedera.pbj.runtime.ProtoWriterTools#writeTag method public.
>
>Ideally move ProtoConstants to being a enum

**Comment by LLITCHEV** [comment]
@jasperpotts I was looking at the other issue (these has two) and it sounded loike there are no methods that allow to write tags. Let me look into this...

**Comment by LLITCHEV** [comment]
Made the changes specified in the first issue.

File: pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/DataTest.java [hunk 1]
```diff
@@ -82,6 +82,10 @@ static Stream<Long> unsignedIntsTestCases() {
         return Stream.of(0,1,9,51,127,Integer.MAX_VALUE*2L).map(Number::longValue);
     }
 
+    static Stream<Integer> unsignedIntsTestCasesInts() {
+        return Stream.of(0,1,9,51,127,(int)Integer.MAX_VALUE*2L).map(Number::intValue);
+    }
```
Comment (artemananiev) [inline pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/DataTest.java:87]: Is this related to the fix?

**Review by artemananiev — DISMISSED** [review]

**Comment by artemananiev** [comment]
Are the newly exposed public methods covered with unit tests?

File: pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/DataTest.java (hunk 1 above)
Comment (LLITCHEV) [reply pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/DataTest.java:87]: This was just a test that was missing - there were for these, but I'll take it out. Thabnks! Good catch!

**Comment by LLITCHEV** [comment]
No, I have taken the newly exposed methods out (the first commit) per conversation with @jasperpotts. There was a mix up in thinking that the 2 issues are the same. This PR only exposes the types of tag values to the otside world. Exposing the new methods will come shortly.

**Comment by artemananiev** [comment]
Closing this PR as superseded by https://github.com/hashgraph/pbj/pull/108.
